x-pack/plugins/infra/public/containers/logs/log_analysis/api/ml_cleanup.ts

```diff
@@ -5,21 +5,25 @@
  */
 
 import * as rt from 'io-ts';
-import { pipe } from 'fp-ts/lib/pipeable';
-import { fold } from 'fp-ts/lib/Either';
-import { identity } from 'fp-ts/lib/function';
-import { npStart } from '../../../../legacy_singletons';
+import type { HttpSetup } from 'src/core/public';
```

Comment:
Yeah it makes sense. I copied the type definition [from](https://github.com/elastic/kibana/blob/5bcf2c8b892d0b456b0dcde18cafc3a401013e9e/x-pack/plugins/infra/public/containers/logs/log_source/api/fetch_log_source_configuration.ts#L16) [other](https://github.com/elastic/kibana/blob/671715f330d49eec3e5b677d53f03e149ee2ac75/x-pack/plugins/infra/public/containers/logs/log_source/log_source.ts#L34) [places](https://github.com/elastic/kibana/blob/5bcf2c8b892d0b456b0dcde18cafc3a401013e9e/x-pack/plugins/infra/public/containers/logs/log_source/api/fetch_log_source_configuration.ts#L16). Since I'm doing it I can also change it there

Reply:
thank you!